.buildkite/pipeline.yml

```diff
@@ -1,15 +1,38 @@
 steps:
-  - label: ":golang: Build"
-    command: "cd src && go build ."
+  - label: ":broom: Lint"
+    key: lint
+    command: "make lint"
     plugins:
-      - golang#v2.0.0:
-          version: 1.21
-          import: github.com/buildkite/agent
+      docker-compose#v4.8.0:
+        run: go
```

Comment:
do all of the steps just use the `go` container in the docker-compose file now? if so we could maybe even just use the plain docker plugin?

Reply:
oh i see there is the plugin tester container isn't plumbed in